examples/java-matter-controller/java/src/com/matter/controller/commands/common/CommandManager.java

```diff
@@ -89,10 +89,10 @@ public final void run(String[] args) {
       }
     }
 
-    String[] temp = Arrays.copyOfRange(args, 1, args.length);
+    String[] temp = Arrays.copyOfRange(args, 2, args.length);

```

Comment:
A comment of `skip over binary and command name` is probably useful here.

Reply:
done, thanks